mabar-frontend/src/services/aiConversationManager.ts

```
@@ -0,0 +1,406 @@
+          const venues = await VenueService.searchVenues(parameters)
+          response = { venues, totalResults: venues?.length || 0 }
+          break
+          
+        case 'findPlayers':
+          const players = await PlayerService.searchPlayers(parameters)
+          response = { players, totalResults: players?.length || 0 }
+          break
+          
+        case 'findSessions':
+          const sessions = await SessionService.searchSessions(parameters)
+          response = { sessions, totalResults: sessions?.length || 0 }
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)